packages/env/lib/build-docker-compose-config.js

```diff
@@ -192,7 +202,7 @@ module.exports = function buildDockerComposeConfig( config ) {
 						dbEnv.credentials.WORDPRESS_DB_PASSWORD,
 					MYSQL_DATABASE: dbEnv.development.WORDPRESS_DB_NAME,
 				},
-				volumes: [ 'mysql:/var/lib/mysql' ],
+				healthcheck,
```

Comment:
I don't think we can delete the `volumes` field because the database is reset every time we restart the environment.